docs/DeveloperGuide.md

```diff
@@ -302,11 +302,222 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
   Use case ends.
 
-* 3a. The given index is invalid.
+* 3a. The given index is out of range
 
-    * 3a1. AddressBook shows an error message.
+    * 3a1. ContaX shows an error message indicating that the given index is invalid.
 
-      Use case resumes at step 2.
+    * Use case resumes at step 2.
+
+* 3b. The given index is of invalid format
+
+    * 3b1. ContaX shows an error message indicating that the given command is of invalid format.
+
+    * Use case resumes at step 2.
```

Comment:
I think can do
```suggestion
* a. The command given is in an invalid format.

    * 3b1. ContaX shows an error message indicating that the given command is of invalid format.

    * Use case resumes at the step before.
```

Reply:
Fixed sentence structure.
Use case resume will specify the MSS step number.